lib/generators/gql/templates/model_type.rb

```diff
@@ -1,7 +1,7 @@
 module Types
-  class <%= @model_name %>Type < Types::BaseObject
```

Comment:
Same question here on the `Type` suffix.

Reply:
Same as above -- it should generate the `Type` suffix by default but I'm good with an option to opt-out of it.